dags/glam.py

```diff
@@ -75,7 +75,7 @@
     destination_table="latest_versions",
     dataset_id=dataset_id,
     project_id=project_id,
-    owner="msamuel@mozilla.com",
+    owner="akommasani@mozilla.com",
```

Comment:
Are these actually necessary? Should this information be inherited from `default_args`?

Reply:
Good call, I'll clean that up and hold on with merging until today's runs complete.